backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5753660_sys_C_Payment_RefundStatus.sql

```diff
@@ -0,0 +1,245 @@
+;
+
+-- 2025-05-06T13:12:19.015Z
+UPDATE AD_Reference_Trl SET Description='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.', Help='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.', IsTranslated='Y',Updated=TO_TIMESTAMP('2025-05-06 16:12:19.006','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Reference_ID=541943
+;
+
+-- Name: Refund Status
+-- 2025-05-06T13:12:32.067Z
+UPDATE AD_Reference SET Help='Indicates the refund processing status of the payment.',Updated=TO_TIMESTAMP('2025-05-06 16:12:32.065','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Reference_ID=541943
+;
+
+-- 2025-05-06T13:12:32.068Z
+UPDATE AD_Reference_Trl trl SET Help='Indicates the refund processing status of the payment.' WHERE AD_Reference_ID=541943 AND AD_Language='de_DE'
+;
+
+-- 2025-05-06T13:12:36.630Z
+UPDATE AD_Reference_Trl SET Description='Indicates the refund processing status of the payment.', Help='Indicates the refund processing status of the payment.', IsTranslated='Y',Updated=TO_TIMESTAMP('2025-05-06 16:12:36.629','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Reference_ID=541943
+;
+
+-- 2025-05-06T13:12:52.782Z
+UPDATE AD_Reference_Trl SET Description='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.', Help='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.', IsTranslated='Y',Updated=TO_TIMESTAMP('2025-05-06 16:12:52.78','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Reference_ID=541943
+;
+
+-- 2025-05-06T13:12:52.783Z
+UPDATE AD_Reference SET Description='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.', Help='Gibt den Status der Rückerstattungsbearbeitung der Zahlung an.' WHERE AD_Reference_ID=541943
+;
+
+-- Column: C_Payment.RefundStatus
+-- 2025-05-06T13:13:22.577Z
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Reference_Value_ID,AD_Table_ID,ColumnName,Created,CreatedBy,DDL_NoForeignKey,EntityType,FacetFilterSeqNo,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsExcludeFromZoomTargets,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRestAPICustomColumn,IsSelectionColumn,IsShowFilterIncrementButtons,IsShowFilterInline,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,589929,583608,0,17,541943,335,'RefundStatus',TO_TIMESTAMP('2025-05-06 16:13:22.348','YYYY-MM-DD HH24:MI:SS.US'),100,'N','D',0,250,'Y','N','Y','N','N','N','N','N','N','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','Y','N',0,'Rückerstattungsstatus',0,0,TO_TIMESTAMP('2025-05-06 16:13:22.348','YYYY-MM-DD HH24:MI:SS.US'),100,0)
```

Comment:
pls, make the filed read-only
Even when is draft, the field shall not be selectable